content/blog/2022/12/camel4roadmap/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Roadmap to Camel v4"
+date: 2022-12-16
+draft: false
+authors: [davsclaus]
+categories: ["Roadmap"]
+preview: "Roadmap to Apache Camel v4 and impact on Camel v3"
+---
+
+Apache Camel v4 is on the way for 1st half in 2023.
+
+The need for Camel v4 is mainly driven by Java open source projects migrating from `javax` to `jakarta` APIs
+and to keep up with popular runtimes such as Spring Boot and Quarkus.
+
+## Primary Goals 
+1. Migrate from `javax` -> `jakarta` (JEE 10)
+2. Java 17 as minimum
+3. Spring Framework 6
+4. Spring Boot 3
+5. Quarkus 3
+
+## Release Goals
+6. Release only what is ready (JEE10 / Java17)
+   This means that Camel components that are not ready (yet) will be dropped in a release until they are ready.
+7. Release Camel Core + Camel Spring Boot together
+8. Move Camel Karaf to Apache Karaf as karaf-camel sub-project 
```

Comment:
Is it official? It has been approved by the 2 PMCs?

Reply:
Not yet, but both the PMCs shows interest in this. The blog post is not written in stone, so we could likely update it, change it or create a new blog post with updates.